build

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Build steps to validate the ketohub raw input spider.
+
+# Run unit tests and calculate code coverage.
+coverage run \
+  --source ketohub \
+  -m unittest discover &&
+# Check that source has correct formatting.
+yapf --diff --recursive --style google ketohub/spiders/*.py &&
```

Comment:
YAPF changed behavior so that it returns 0 even when there are diffs, so we have to use a workaround:

https://github.com/JeetShetty/GreenPiThumb/blob/master/build#L18...L29

They *just* changed this, but the change isn't released yet:

https://github.com/google/yapf/commit/eeb6d46981bb9d51c9ea1cecc16ec73a03ad7d4c